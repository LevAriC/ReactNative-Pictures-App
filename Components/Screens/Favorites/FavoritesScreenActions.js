import { ADD_TO_FAVORITES } from './FavoritesScreenActionTypes'

const addToFavorites = (pic) => ({
    type: ADD_TO_FAVORITES,
    data: { pic }
})

const handleAddToFavorites = (pic) => async dispatch => {
    console.log('handleAddToFavorites')
    dispatch(addToFavorites(pic))
}

export default {
    handleAddToFavorites
}