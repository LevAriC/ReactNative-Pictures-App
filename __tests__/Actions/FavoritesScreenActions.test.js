import FavoritesScreenActions from '../../Components/Screens/Favorites/FavoritesScreenActions'

it("Testing Action's type", () => {
    expect(FavoritesScreenActions.addToFavorites().type).toMatchSnapshot();
    expect(FavoritesScreenActions.changeView().type).toMatchSnapshot();
    expect(FavoritesScreenActions.saveBigPic().type).toMatchSnapshot();
});