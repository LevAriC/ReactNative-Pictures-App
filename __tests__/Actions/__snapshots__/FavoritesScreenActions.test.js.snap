// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Action's type 1`] = `"ADD_TO_FAVORITES"`;

exports[`Testing Action's type 2`] = `"CHANGE_VIEW"`;

exports[`Testing Action's type 3`] = `"SAVE_BIG_PIC"`;
